### PR TITLE
UPGRADE FOUNDATION ON SJ DEMO WEBSITE

### PR DESCRIPTION
**Pull request by paul-mesnilgrente**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+// blocks
+@import 'blocks/add_to_set';
+@import 'blocks/category_tabs';
+@import 'blocks/filter_container';
+@import 'blocks/flash';
+@import 'blocks/footer';
+@import 'blocks/gallery';
+@import 'blocks/home';
+@import 'blocks/records';
+@import 'blocks/top-bar';
```
Comment (katherine-boost) [inline app/assets/stylesheets/application.scss]: Not all of these seem to be blocks? some have a few different things in them and some are more Foundation modifications than blocks. But I think it makes sense to arrange them in here, ready for future refactoring 👍

File: app/assets/stylesheets/blocks/_flash.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.callout {
+  &.notice {
+    @include callout-style(map-get($foundation-palette, success));
+  }
+}
```
Comment (katherine-boost) [inline app/assets/stylesheets/blocks/_flash.scss]: In Fincap, we have Foundation stuff like this under `app/assets/stylesheets/foundation`. I'd be tempted to do something similar here, to make clear things are completely custom styles and which are doing Foundation mods. Something to ponder

File: app/assets/stylesheets/blocks/footer.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+footer {
+  border-top: 1px solid $dark-gray;
+
+  a {
+    color: $dark-gray;
+    line-height: 2rem;
+  }
+}
```
Comment (katherine-boost) [inline app/assets/stylesheets/blocks/footer.scss]: Another "In Fincap ...." comment 😉We have things like this in a `_base.scss` file, which I think is quite good for element-level styles.

Our structure might be worth a look for you: https://github.com/boost/fincap-client-voices/tree/master/app/assets/stylesheets

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -1,14 +1,28 @@
...
- *
- * You're free to add application-wide styles to this file and they'll appear at the top of the
- * compiled file, but it's generally better to create a new file per style scope.
- *
- *= require foundation_and_overrides
- *= require_tree .
- *= require_self
- */
+// fonts
+@import 'fonts/railway';
```
Comment (katherine-boost) [inline app/assets/stylesheets/application.scss]: I think it's Raleway, no? https://fonts.google.com/specimen/Raleway

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (paul-mesnilgrente) [reply app/assets/stylesheets/application.scss]: Yes! Something to sort out in the future :)

File: app/assets/stylesheets/blocks/_flash.scss (hunk 2 above)
Comment (paul-mesnilgrente) [reply app/assets/stylesheets/blocks/_flash.scss]: Quite agree on the idea. I don't have time for checking what's overriding Foundation styles though :/

File: app/assets/stylesheets/application.scss (hunk 4 above)
Comment (paul-mesnilgrente) [reply app/assets/stylesheets/application.scss]: Good spotting! Fixed

File: app/assets/stylesheets/blocks/_flash.scss (hunk 2 above)
Comment (katherine-boost) [reply app/assets/stylesheets/blocks/_flash.scss]: Fair enough! It's too much to unravel right now. 

I recognised `.callout`, `.tabs-content`, and `.top-bar` as Foundation classes.

File: app/assets/stylesheets/blocks/footer.scss (hunk 3 above)
Comment (paul-mesnilgrente) [reply app/assets/stylesheets/blocks/footer.scss]: Good point, changed it :)

**Pull request merged**